android/app/src/main/res/drawable/music_player_widget_button_bar_background.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android" android:shape="rectangle">
+    <solid android:color="@color/widget_button_bar_background"/>
+    <corners android:radius="16dp"/>
```

Comment:
<img width="529" alt="image" src="https://github.com/user-attachments/assets/096ad4c4-0264-440b-9959-311a35b3494b">

We should make it equal to <widget radius> - <padding> to make it look perfectly even

Very nuanced, but I know that it triggers some people when it's not

Reply:
Alternatively we can reduce the widget padding itself

Reply:
Fixed.